Alignment/include/ActsAlignment/Kernel/AlignmentMask.hpp

```diff
@@ -0,0 +1,39 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2019 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Utilities/EnumBitwiseOperators.hpp"
+
+#include <limits>
+#include <type_traits>
+
+namespace ActsAlignment {
+
+/// Collection of bit masks to enable steering which alignment degree of freedom
```

Comment:
For my education: will we do/plan something like this also for the levels?
i.e. masks for CenterStave, CenterLayer, CenterDetector

Reply:
Yes, but I suppose those maks could be used for any geometry object?

Reply:
True, I just wonder how we can introduce some hierarchy into the system - anyhow, let's follow this up in a future PR.